templates/ep19/bs/base.html

```diff
@@ -20,9 +20,9 @@
     {% endblock meta %}
 
     {# TODO: make it local link to boostrap #}
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-    <link rel="stylesheet" href="/static/css/ep2019-theme.css">
-    <link rel="stylesheet" href="/static/css/ep2019-customisations.css">
+    <link rel="stylesheet" href="{% static 'css/bootstrap.min.css' %}">
+    <link rel="stylesheet" href="{% static 'css/ep2019-theme.css' %}">
+    <link rel="stylesheet" href="{% static 'css/ep2019-customisations.css' %}">
```

Comment:
can we merge the last two css files?

Reply:
`ep2019-theme` seems like a minified bootstrap (possibly just a subsection, the comments don't say - @artcz do you know?) the other seems to be manually-maintained file. We could probably retire either `ep2019-theme.css` completely, seems like it contains an older bootstrap version and is generally a smaller file
```
-rw-r--r-- 1 jakub jakub 155758 May 18 19:54 bootstrap.min.css
-rw-r--r-- 1 jakub jakub   3091 May  8 18:19 bs-stepper.min.css
-rw-r--r-- 1 jakub jakub   3211 May  8 18:19 ep2019-customisations.css
-rw-r--r-- 1 jakub jakub 121721 May  8 18:19 ep2019-theme.css
```

Reply:
Yeah, ep-2019 seems safe to remove. Thanks for pointing it out!